src/chocolatey/infrastructure.app/services/IFilesService.cs

```diff
@@ -16,10 +16,11 @@
 
 namespace chocolatey.infrastructure.app.services
 {
+    using System;
+    using System.IO;

```

Comment:
Where did the need for this addition come from?

Reply:
The `System` using isn't new, it was just correctly ordered.

The `System.IO` was automatically inserted, so I honestly don't know. I'll get it removed if it isn't needed (which it doesn't look like it is).